### ISSUE TITLE
27. App info modal

### ISSUE THREAD

**Issue by eonflower** [the-collab-lab/tcl-66-smart-shopping-list#61]
## Description

Add a button to the nav bar that when clicked, opens a modal with more info about the app.

## Acceptance Criteria

- [x] It should be styled to match the app
- [x] A modal should pop up
- [x] Add a description of how to use the app
- [x] Give user basic understanding of app

## Notes

Potentially add some credits / links
